src/app/components/listing/details/surveillance/nonconformity/inspect.html

```diff
@@ -47,6 +45,11 @@ <h4 class="modal-title" id="inspect-label">Inspect Nonconformities</h4>
               <td ng-if="noncon.capMustCompleteDate">{{ noncon.capMustCompleteDate | date : 'mediumDate' : 'UTC' }}</td>
               <td ng-if="!noncon.capMustCompleteDate">N/A</td>
             </tr>
+            <tr>
+              <th scope="row">Non-conformity closed date</th>
```

Comment:
Should this be 
```suggestion
              <th scope="row">Non-conformity close date</th>
```
to match the tense of the variable?